src/renderer/helpers/api/invidious.js

```diff
@@ -0,0 +1,131 @@
+import store from '../../store/index'
+import { stripHTML, toLocalePublicationString } from '../utils'
+import autolinker from 'autolinker'
+
+function getCurrentInstance() {
+  return store.getters.getCurrentInvidiousInstance
+}
+
+export function invidiousAPICall({ resource, id = '', params = {} }) {
+  return new Promise((resolve, reject) => {
+    const requestUrl = getCurrentInstance() + '/api/v1/' + resource + '/' + id + '?' + new URLSearchParams(params).toString()
+
+    fetch(requestUrl)
+      .then((response) => response.json())
+      .then((json) => {
+        if (json.error !== undefined) {
+          throw new Error(json.error)
+        }
+        resolve(json)
+      })
+      .catch((error) => {
+        console.error('Invidious API error', requestUrl, error)
+        reject(error)
+      })
+  })
+}
+
+export async function invidiousGetChannelInfo(channelId) {
+  return new Promise((resolve, reject) => {
+    invidiousAPICall({
+      resource: 'channels',
+      id: channelId,
+    }).then(response => {
+      resolve(response)
+    }).catch(xhr => {
+      console.error(xhr)
+      reject(xhr)
+    })
+  })
```

Comment:
As the error is already logged in `invidiousApiCall` and in the error handling calling function, this function can be simplified to this.
```suggestion
  return await invidiousAPICall({
    resource: 'channels',
    id: channelId,
  })
```